truffle/src/main/java/org/jruby/truffle/RubyEngine.java

```diff
@@ -35,6 +36,7 @@ public RubyEngine(RubyInstanceConfig instanceConfig) {
 
         engine = PolyglotEngine.newBuilder()
                 .config(RubyLanguage.MIME_TYPE, INSTANCE_CONFIG_KEY, instanceConfig)
+                .config(RubyLanguage.MIME_TYPE, OptionsCatalogue.ARGUMENTS.getName(), instanceConfig.getArgv())
```

Comment:
Ah, but options can just not be passed and then use the defaults I guess, except from some key things like ARGV coming from non-system property or OptionDescriptor defaults.